### PR TITLE
Added Senior Cargo Technician

### PR DESCRIPTION
**Pull request by InfinityPandaRed**
## About the PR

added senior clothes and pda that can be unlocked after a certain amount of cargo playtime
## Why / Balance

every major department has a senior role except cargo. WHY NOT
## Media

image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

- added senior cargo jumpsuit
- added senior cargo softcap
- added senior cargo PDA and ID

## Release Notes

- **New Features**
  - Introduced automated changelogs for better tracking of updates.
  - Added fire fighting remotes in atmos lockers.
  - New lobby music track and the "Living Heart" item for heretics.
  - Added HECU operative job/loadout and the new species Rodentia.
  - Enhanced character customization with a preferred borg name field.
  - New loadout options for senior cargo technicians, including uniforms and equipment.

- **Bug Fixes**
  - Resolved various gameplay issues, including item sprite problems and character abilities.

- **Improvements**
  - Adjustments to item pricing, weapon rebalancing, and character behaviors for a refined gameplay experience.

### CODE REVIEW

**Comment by whateverusername0** [comment]
magic :trollface:

File: Resources/Prototypes/Loadouts/loadout_groups.yml [hunk 1]
```diff
@@ -606,6 +606,7 @@
   loadouts:
   - CargoTechnicianHead
   - CargoTechnicianHeadMail # Goob/DeltaV - Mail
+  - CargoTechnicianHeadSenior # Goob/DeltaV - Senior
```
Comment (whateverusername0) [inline Resources/Prototypes/Loadouts/loadout_groups.yml:609]: why the /DeltaV? we're not furrystation ( :trollface: )

File: Resources/Locale/en-US/preferences/loadout-groups.ftl [hunk 2]
```diff
@@ -98,6 +98,7 @@ loadout-group-cargo-technician-jumpsuit = Cargo Technician jumpsuit
 loadout-group-cargo-technician-backpack = Cargo Technician backpack
 loadout-group-cargo-technician-outerclothing = Cargo Technician outer clothing
 loadout-group-cargo-technician-shoes = Cargo Technician shoes
+loadout-group-cargo-technician-id = Cargo Technician ID
```
[the remark this answers is not in the document]
Comment (Piras314) [reply Resources/Locale/en-US/preferences/loadout-groups.ftl:101]: this should be in a goob file not with a comment

**Comment by Piras314** [comment]
yaml linter error: Duplicate ID: 'CargoTechnicianJumpsuitMail'

**Comment by Piras314** [comment]
conflicts & yaml linter is still not fixed @InfinityPandaRed

**Comment by Aidenkrz** [comment]
> magic :trollface:

this guys is fucking stupid and doesnt know how to resolve merge conflicts

**Comment by a-person5660** [comment]
CL should be done like:

Changelog
🆑

    tweak:

**Comment by Aidenkrz** [comment]
Derelict PR

**Pull request closed without merging**